### ISSUE TITLE
Nice Idea

### ISSUE THREAD

**Issue by rohybnol** [charliewolfe/Stealthy-Kernelmode-Injector#2]
Firstly thanks for the neat ideas in that project which you share publicly,  I tried it out and it works flawlessly for a messagebox sample.
I tried a sample dll from me, however it seems that it is not possible to create a thread inside when choosing your execution over hook?
Is it needed to go over the SetWindowsHooks and a hook in general, or would it be possible to call the dll entry point itself oldshool?

TLDR:
I am not able to get other dlls to properly inject with your injector for everything else then a sample message box and I am asking why, as there is no crash or something, just the execution stucks which I guess is cause of the CreateThread in the dll.

btw I am on 21H2 but the offsets and patterns seems fine too
